views/js/qtiCreator/test/MathEditor/test.js

```diff
@@ -0,0 +1,67 @@
+define([
+    'jquery',
+    'taoQtiItem/qtiCreator/editor/MathEditor'
+],
+function($, MathEditor) {
+    'use strict';
+
+    QUnit.asyncTest('Latex rendering', function test(assert) {
```

Comment:
We need all tests cases to be wrapped under a module (for reporting), something like : 

`QUnit.module('MathEditor rendering');`